spec

```diff
@@ -109,6 +109,9 @@ beginseg
     include "build/src/libultra/os/gethwintrroutine.o"
     include "build/asm/__osSetWatchLo.o"
     include "build/data/rsp_boot.text.o"
+#ifdef COMPILER_GCC
+    include "build/src/gcc-fix/missing_gcc_functions.o"
```

Comment:
huge nitpick here but this is inconsistent `-`/`_` usage, may as well fix it before it's too late